tasks/src/main/scala/dagr/tasks/picard/MergeBamAlignment.scala

```diff
@@ -42,11 +42,13 @@ class MergeBamAlignment(unmapped: PathToBam,
                         programGroupCommand: Option[String] = None,
                         programGroupName: Option[String] = None,
                         attributesToRetain: List[String] = List[String]("X0", "ZS", "ZI", "ZM", "ZC", "ZN"),
+                        attributesToReverse: List[String] = List[String]("ad", "bd", "cd", "ae", "be", "ce"), // fgbio consensus tags
```

Comment:
IT would be nice to put these in an `Fgbio` object or similar in case we want to reference them in another task.  I just don't want to forget about them here.